### DISCUSSION TITLE
Which parameters are modified to bulid renal insufficiency model?

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#840] [Questions & Problems] [answered]
Dear all,
 I want to build a model of renal insufficiency. Does anyone know which literature describes the changes in kidney volume, blood flow, gastric emptying time and other related physiological anatomical parameters in different stages of renal insufficiency?
Yours,
Sincerely

**Comment by prvmalik** [comment, marked as answer]
[CKD 2020.pdf](https://github.com/Open-Systems-Pharmacology/Forum/files/5276860/CKD.2020.pdf)
I am attaching a preprint of an article accepted for publication in the Journal of Clinical Pharmacology, which details the creation and evaluation of a CKD population in PKSim.

**Comment by Wenjing1018** [comment]
Dear prvmalik ,
 The paper you provide is of great help to me, but now I can' t check it on the website. So,I would appreciate it if you sent me the supplementary material of this paper.Thank you very much!
Best wishes.

**Comment by Zhaowenyu-gaga** [comment]
Dear Wenjing,

你好 ，在论坛里看到很多您有关这个软件的提问，我不是药学专业的，但是想应用这个软件做一下某有机化合物在人体中的代谢状况的模拟，建成PBPK模型，请问这个软件的教程可以去哪里学习，之前完全没有接触过此类软件，如果您能回答，我将非常的感谢！

**Comment by Wenjing1018** [comment]
OSP官网有软件说明书、教程，都可以学习，你多翻翻他们的官网。


| |
潘文静
|
|
13003036211@163.com
|
签名由网易邮箱大师定制
On 11/2/2020 21:40，Zhaowenyu-gaga<notifications@github.com> wrote：

Dear Wenjing,

你好 ，在论坛里看到很多您有关这个软件的提问，我不是药学专业的，但是想应用这个软件做一下某有机化合物在人体中的代谢状况的模拟，建成PBPK模型，请问这个软件的教程可以去哪里学习，之前完全没有接触过此类软件，如果您能回答，我将非常的感谢！

**Comment by Zhaowenyu-gaga** [comment]
感谢回复，我后面好好读一下。


我这个想法可以使用这个软件进行拟合吧？看起来好像没有你们做药物的那么复杂，也没有血药浓度什么的这种，目前只是想最后得到一个模型可以：拟合该化合物在不同时间，不同摄入浓度在体内（哺乳动物或者家禽）各个组织中的代谢情况，这个软件是否适用？


夜间打扰，您有空时候回我就行，非常感谢。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Open-Systems-Pharmacology/Forum"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年11月2日(星期一) 晚上9:50
收件人:&nbsp;"Open-Systems-Pharmacology/Forum"<Forum@noreply.github.com&gt;;
抄送:&nbsp;"赵文玉"<1037220832@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [Open-Systems-Pharmacology/Forum]  Which parameters are modified to bulid renal insufficiency model? (#614)





 OSP官网有软件说明书、教程，都可以学习，你多翻翻他们的官网。
 
 
 | |
 潘文静
 |
 |
 13003036211@163.com
 |
 签名由网易邮箱大师定制
 On 11/2/2020 21:40，Zhaowenyu-gaga<notifications@github.com&gt; wrote：
 
 Dear Wenjing,
 
 你好 ，在论坛里看到很多您有关这个软件的提问，我不是药学专业的，但是想应用这个软件做一下某有机化合物在人体中的代谢状况的模拟，建成PBPK模型，请问这个软件的教程可以去哪里学习，之前完全没有接触过此类软件，如果您能回答，我将非常的感谢！

**Comment by StephanSchaller** [comment]
Please write in English so other users might help you
